### PR TITLE
Ania c

### PR DESCRIPTION
**Pull request by annolando**
wrong navbar

### CODE REVIEW

File: apps/bootstrap/AniaC/index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+              <li class="nav-item">
+                <a class="nav-link" href="#">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Pricing</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link disabled">Disabled</a>
+              </li>
+            </ul>
```
Comment (apietryga) [inline apps/bootstrap/AniaC/index.html:34]: <form class="d-flex" role="search">
          <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
          <button class="btn btn-outline-success" type="submit">Search</button>
        </form>

File: apps/bootstrap/AniaC/index.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          </button> 
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Features</a>
+              </li>
+              <li class="nav-item">
```
Comment (apietryga) [inline apps/bootstrap/AniaC/index.html]: cały li z pricing niepotrzebny

File: apps/bootstrap/AniaC/style.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.carousel-item img{
+    height:32rem;
+}
+.carousel.item h5{
```
Comment (apietryga) [inline apps/bootstrap/AniaC/style.css]: carousel-item

File: apps/bootstrap/AniaC/style.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    color: white;
+    font-size: 3rem;
+    font-weight: bold;
+}
+.carousel.item p{
```
Comment (apietryga) [inline apps/bootstrap/AniaC/style.css]: carousel-item

File: apps/bootstrap/AniaC/style.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    color:beige;
+    font-size: 1.5rem;
+
+}
+.carousel.item:nth-child(1)h5, .carousel.item:nth-child(1) p{
```
Comment (apietryga) [inline apps/bootstrap/AniaC/style.css]: carousel-item

File: apps/bootstrap/AniaC/style.css [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    
+    text-align: right;
+
+}
+.carousel.item:nth-child(3)h5, .carousel.item:nth-child(3) p{
```
Comment (apietryga) [inline apps/bootstrap/AniaC/style.css]: carousel-item

**Review by apietryga — CHANGES_REQUESTED** [review]
ogólnie super ! 👍🏼

File: apps/bootstrap/AniaC/index.html (hunk 1 above)
Comment (annolando) [reply apps/bootstrap/AniaC/index.html:34]: done

File: apps/bootstrap/AniaC/index.html (hunk 2 above)
Comment (annolando) [reply apps/bootstrap/AniaC/index.html]: done

File: apps/bootstrap/AniaC/style.css (hunk 4 above)
Comment (annolando) [reply apps/bootstrap/AniaC/style.css]: done

File: apps/bootstrap/AniaC/style.css (hunk 6 above)
Comment (annolando) [reply apps/bootstrap/AniaC/style.css]: done

**Comment by micszy79** [comment]
good job

**Review by apietryga — APPROVED** [review]

**Pull request merged**